interfaces/IBF-dashboard/src/app/components/chat/chat.component.html

```diff
@@ -19,64 +19,11 @@
     ></app-about-btn>
     <app-video-guide-button color="ibf-white"></app-video-guide-button>
   </app-dialogue-turn>
-  <!-- New event today -->
+  <!-- Active event -->
   <app-dialogue-turn
     *ngIf="
       eventService.state.activeEvent &&
-      eventService.state.activeTrigger &&
-      eventService.state.newEvent
-    "
-  >
-    <p>
-      <span
-        [innerHTML]="
-          'chat-component.' +
-            disasterCategory +
-            'new-event-active-trigger.welcome'
-            | translate
-              : {
-                  name: authService.displayName,
-                  leadTime: eventService.state.triggerLeadTime,
-                  timeUnit: eventService.state.timeUnit,
-                  disasterTypeLabel: disasterTypeLabel,
-                  monthYear: eventService.state.firstLeadTimeName
```

Comment:
@SwatiTh 
- the {{startDate}} bug was startDate was missing here as a parameter that needed to be filled in
- the reason you did see it at first, but not when you checked again at a later date, is because of the distinction between 'new-event-active-trigger' and 'active-event-active-trigger', where the current date matters in relation to the event start date. And for 'active-event-active-trigger', startDate was correctly added (see below).
- Because additionally I found that both texts were now identical anyway, I decided to remove the distinction altogether. This simplifies the code.